lib/retrieval-stats.js

```diff
@@ -92,15 +92,15 @@ export const buildRetrievalStats = (measurements, telemetryPoint) => {
  * @param {string} fieldNamePrefix
  * @param {number[]} values
  */
-const addHistogramToPoint = (point, fieldNamePrefix, values) => {
+const addHistogramToPoint = (point, values, fieldNamePrefix = '') => {
```

Comment:
why was this changed? Iiuc we never pass an empty prefix

Reply:
It did pass an empty prefix in the first iteration; see [658cb1c](https://github.com/filecoin-station/spark-evaluate/pull/97/commits/658cb1ced84718babfe090eb0a94c097991140e3).

I think this can be useful again in the future, so I kept it.